packages/opentelemetry-node/test/NodeTracerProvider.test.ts

```diff
@@ -84,26 +84,27 @@ describe('NodeTracerProvider', () => {
     it('should load a merge of user configured and default plugins and implictly enable non-default plugins', () => {
       provider = new NodeTracerProvider({
         logger: new NoopLogger(),
-        plugins: {
-          'simple-module': {
-            path: '@opentelemetry/plugin-simple-module',
-          },
-          'supported-module': {
-            path: '@opentelemetry/plugin-supported-module',
-            enhancedDatabaseReporting: false,
-            ignoreMethods: [],
-            ignoreUrls: [],
-          },
-          'random-module': {
-            enabled: false,
-            path: '@opentelemetry/random-module',
-          },
-          http: {
-            path: '@opentelemetry/plugin-http-module',
-          },
-        },
+        // plugins: {
```

Comment:
why is this commented ?

Reply:
As @dyladan suggested, I made this quick prototype to show how the new design for plugins will work out and it's not meant to be merged. I commented the failing tests for now to be able to compile the code and test the plugins on a sample project. Once the design is approved I'll update the tests and also add new ones.